src/components/dialogs/parameters/parameters.js

```diff
@@ -81,32 +80,27 @@ import {
 } from '../../utils/optional-services';
 import { useOptionalServiceStatus } from '../../../hooks/use-optional-service-status';
 
-export const CloseButton = ({ hideParameters, classeStyleName }) => {
+export const CloseButton = ({ hideParameters, ...props }) => {
     return (
-        <LabelledButton
-            callback={hideParameters}
-            label={'close'}
-            name={classeStyleName}
-        />
+        <LabelledButton callback={hideParameters} label={'close'} {...props} />
     );
 };
```

Comment:
We lost the `styles.button` passed from children who isn't neither passed to `<LabelledButton/>`
```diff
- <CloseButton hideParameters={hideParameters} className={classes.button} />
+ <CloseButton hideParameters={hideParameters} />
```